includes/wikia/api/DWDimensionApiController.class.php

```diff
@@ -52,11 +52,16 @@ public function getWikiDartTags() {
 
 		$result = [];
 		while ($row = $db->fetchObject($dbResult)) {
-			$result[] = [
-				'wiki_id' => $row->cv_city_id,
-				'tag' => null,
-				'value' => $row->cv_value
-			];
+			#extract from list like "s:199:\"sex=m;sex=f;age=under18;age=13-17;age=18-24;age=25-34;age=18-34;\";"
+			preg_match_all("/([^;= ]+)=([^;= ]+)/", unserialize( $row->cv_value ), $r);
+
+			for ($i = 0; $i < count( $r[1] ); $i++) {
+				$result[] = [
+					'wiki_id' => $row->cv_city_id,
+					'tag' => $r[ 1 ][ $i ],
+					'value' => $r[ 2 ][ $i ]
+				];
+			}
```

Comment:
Don't we want to move WikiFactory to a service. This add a dependency on a database. Let's use `WikiFactory` class helpers to get a variable value. It's pretty well cached.

Reply:
WikiFactory in it's current form doesn't have a method to fetch values for all wikis, only a single one (at least from what I have seen). I was thinking about adding one, but wasn't sure if it's worth it. Do you think I should do that? Or is there some existing mechanism that I missed?

Reply:
Ok moved to WikiFactory